docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md

```diff
@@ -24,39 +24,50 @@ YugabyteDB Anywhere allows you to expand a universe to add more nodes and shrink
+    Using the **Edit Universe** page, you can modify the following:
 
-Note that you can't change the replication factor of a universe.
+    - **Regions** - you can select any region configured in the provider used to deploy the universe.
+    - [Master Placement](../../create-deployments/dedicated-master/).
+    - **Total Nodes** and **Availability Zones** - as you add nodes, they are automatically distributed among the availability zones; you can also add, configure, and remove availability zones.
+    - **Instance Configuration** - change instance type and volume size as configured in the provider. In some cases, these operations are available as a [smart resize](#smart-resize).
+    - [User tags](../instance-tags/).
+
+    Note that you can't change the replication factor of a universe.
+
+1. Click **Save**.
+
+YugabyteDB automatically ensures that new nodes start hosting the tablet leaders for a set of tablets in such a way that the tablet leader count remains evenly balanced across all the available nodes.
 
 To change the number of nodes of universes created with an on-premises cloud provider and secured with third-party certificates obtained from external certification authorities, you must first add the certificates to the nodes you will add to the universe. Refer to [Add certificates](../../security/enable-encryption-in-transit/add-certificate-ca/). Ensure that the certificates are signed by the same external CA and have the same root certificate. In addition, ensure that you copy the certificates to the same locations that you originally used when creating the universe.
 
-### Smart resize
+## Smart resize
+
+Normally when resizing a universe, YugabyteDB moves the data from the old nodes to the new nodes. However, if the universe is deployed on AWS, GCP, or Azure using a [cloud provider configuration](../../configure-yugabyte-platform/aws/), you can change the instance type and increase the volume size without migrating the data. This is referred to as smart resize, and can be significantly faster than a full copy of the data.
+
+Smart resize is available for the following operations:
+
+- Change the Instance type.
 
-For universes that use Google Cloud Provider (GCP), Amazon Web Services (AWS), or Microsoft Azure, YugabyteDB Anywhere allows you to change the VM images and increase the volume size without moving the data from the old nodes to the new nodes. This is known as smart resize and is subject to the following:
+    Note that smart resize is not available when changing the instance type from an AWS EBS-backed instance type (like c5.xlarge) to a local storage-backed instance type (like i3.xlarge), or vice-versa.
```

Comment:
minor: local-storage-backed? Not sure about hyphenation convention in English :)